website/common/locales/en/limited.json

```diff
@@ -92,5 +92,8 @@
     "winterEventAvailability": "Available for purchase until January 31.",
     "springEventAvailability": "Available for purchase until May 31.",
     "summerEventAvailability": "Available for purchase until August 31.",
-    "valentinesEventAvailability": "Available for purchase until February 28."
+    "valentinesEventAvailability": "Available for purchase until February 28.",
+    "managerIdRequired": "req.body.managerId is required",
```

Comment:
It should go in the apiMessages file. Some translators would try to convert `req.body.managerId` to their own language.